Makefile

```diff
@@ -1,5 +1,14 @@
-OPT = -std=c++11
-INC = -I/usr/lib/llvm-3.8/include
+UNAME_S := $(shell uname -s)
+
+OPT = -std=c++14
+ifeq ($(UNAME_S),Linux)
+    INC = -I/usr/lib/llvm-11/include
+    CON = llvm-config-11 --ldflags --libs all --system-libs
+endif
+ifeq ($(UNAME_S),Darwin)
```

Comment:
I mean, not everyone uses Brew. Just sayin' ;)

(nix? macports? pkgsrc?)

Please ignore me.

Reply:
Tbh this can't actually compile on mac arm because there are no symbols for some of the things being done, this was just more so for me trying to square out as many issues as possible so it can be removed when everything is working